mkdocs.yml

```diff
@@ -160,6 +160,12 @@ markdown_extensions:
       emoji_generator: !!python/name:material.extensions.emoji.to_svg
 
   - md_in_html
+  # https://facelessuser.github.io/pymdown-extensions/extensions/betterem/
+  - pymdownx.betterem
+  # https://facelessuser.github.io/pymdown-extensions/extensions/caret/
+  - pymdownx.caret # h^2^o
+  # https://facelessuser.github.io/pymdown-extensions/extensions/tilde/
+  - pymdownx.tilde # ~~deleted~~
```

Comment:
👀 where do we use these in the docs?

Reply:
I added this with the hope I can take advantage of these in the future. I often use ~~crossed~~ when I write markdown, and the ability to superscript stuff seems nice (can pull that in with https://github.com/sbrunner/jsonschema2md/pull/405)

Reply:
I can't imagine we'd want crossed-out text in our documentation, but I won't fight this too hard 😂